client/packages/coldchain/src/Equipment/ListView/Toolbar.tsx

```diff
@@ -69,9 +69,7 @@ export const Toolbar = () => {
           value: AssetLogStatusInput.Functioning,
         },
         {
-          label: t('status.functioning-but-needs-attention', {
```

Comment:
Sidequest: Went through and removed all the namespaces to clear console error... sorry 🙏🏻